Sources/GroundState/py_variational_montecarlo.hpp

```diff
@@ -29,9 +29,84 @@ namespace py = pybind11;
+      .def_readonly("error_of_mean", &Stats::error_of_mean)
+      .def_readonly("variance", &Stats::variance)
+      .def_readonly("autocorrelation", &Stats::correlation)
+      .def_readonly("R", &Stats::R);
+
+  py::class_<MCResult>(m_vmc, "MCResult",
+                       R"EOF(Result of Monte Carlo sampling.)EOF")
+      .def_property_readonly(
+          "samples",
+          [](const MCResult &self) {
+            assert(self.samples.rows() % self.n_chains == 0);
+            return detail::as_readonly(py::array_t<double, py::array::c_style>{
+                {self.samples.rows() / self.n_chains, self.n_chains,
+                 self.samples.cols()},
+                self.samples.data(),
+                py::none()});
+          },
+          py::keep_alive<1, 0>{},
+          R"EOF(Visible configurations `{vᵢ}` visited during sampling.
+
+                Visible configurations are represented by a row-major matrix
+                of `float64` where every row is a visible configuration.)EOF")
+      .def_property_readonly(
+          "log_values",
+          [](const MCResult &self) {
+            assert(self.log_values.rows() % self.n_chains == 0);
+            return detail::as_readonly(py::array_t<Complex, py::array::c_style>{
+                {self.log_values.rows() / self.n_chains, self.n_chains},
+                self.log_values.data(),
+                py::none()});
+          },
+          py::keep_alive<1, 0>{},
+          R"EOF(A vector of `complex128` representing `Ψ(vᵢ)` for all
+                    sampled visible configurations `vᵢ`.)EOF")
+      .def_property_readonly(
+          "der_logs",
+          [](const MCResult &self) -> py::object {
+            if (self.der_logs.has_value()) {
+              assert(self.der_logs->rows() % self.n_chains == 0);
+              return detail::as_readonly(
+                  py::array_t<Complex, py::array::c_style>{
+                      {self.der_logs->rows() / self.n_chains, self.n_chains,
+                       self.der_logs->cols()},
+                      self.der_logs->data(),
+                      py::none()});
+            }
+            return py::none();
+          },
+          py::keep_alive<1, 0>{},
+          R"EOF(A matrix of logarithmic derivatives.
+
+                Each row in the matrix corresponds to the gradient of
+                `Ψ(vᵢ)` with respect to variational parameters.)EOF")
+      .def_readonly("n_chains", &MCResult::n_chains,
+                    R"EOF(Number of Markov Chains which this object represents.
+
+                    If `n_chains > 1`, then the first visible configuration
+                    comes from the first Markov Chain, the second -- from the
+                    second Markov chain, etc. The `n_chains + 1`st visible
+                    configuration is again from the first Markov Chains.
+                    Wavefunction logarithms and derivatives are interleaved in a
+                    similar fashion.
```

Comment:
`similar` or `the same`? Writing _similar_ is potentially confusing as it does not make clear, what is the same and what is different.